amz_review_scraper/config.py

```diff
@@ -13,33 +13,44 @@ def create_db_url(user, pw, url, db):
     return f"postgresql://{user}:{pw}@{url}/{db}"
 
 
-# Set environment :: "development" or "production"
-env_setting = "production"
-
-
 # import .env variables for DB connection
-if env_setting == "development":
-    POSTGRES_USER = get_env_variable("DEV_POSTGRES_USER")
-    POSTGRES_PW = get_env_variable("DEV_POSTGRES_PW")
-    POSTGRES_URL = get_env_variable("DEV_POSTGRES_URL")
-    POSTGRES_DB = get_env_variable("DEV_POSTGRES_DB")
-elif env_setting == "production":
-    POSTGRES_USER = get_env_variable("PROD_POSTGRES_USER")
-    POSTGRES_PW = get_env_variable("PROD_POSTGRES_PW")
-    POSTGRES_URL = get_env_variable("PROD_POSTGRES_URL")
-    POSTGRES_DB = get_env_variable("PROD_POSTGRES_DB")
+# TODO: this can probably be simplified more
+def get_env_db_url(env_setting):
+    if env_setting == "development":
```

Comment:
Here I would unify the ENV var names, and then have this just reading from different .env files